app/src/main/java/com/example/intervaltimer/TimerActivity.kt

```diff
@@ -49,16 +54,32 @@ class TimerActivity : AppCompatActivity() {
 
         invalidateOptionsMenu()
 
-        mediaPlayer = MediaPlayer.create(this, R.raw.bellring)
+        workoutName.text = args.intervalList[0].name // placeholder code for setting text at startup
+
+        if (args.intervalList[0].reps.toString() != "null"){
+            repsText.text = args.intervalList[0].reps.toString() // placeholder code for setting text at startup
+        }
+
+        val covertTime: Int? = args.intervalList[0].time
+
+        val minutes = covertTime!! / 60
+        val seconds = covertTime % 60
+
+
+        val covertTimeText = String.format("%02d:%02d", minutes, seconds)
 
-        workoutName.text = "Push ups" // placeholder code for setting text at startup
-        repsText.text = "# of reps 50" // placeholder code for setting text at startup
+        totaltime = (minutes * 60000 + seconds * 1000 + 1000).toLong()
 
-        timerText.text = "00:10" // placeholder code for setting text at startup
+        timerText.text = covertTimeText
+
+        timerText.setTextColor(Color.parseColor(args.intervalList[0].color))
```

Comment:
Pull out `Color.parseColor(args.intervalList[0].color))` to its own variable instead of making 4 separate calls to `parseColor()`